kmeaningful/fit_assign.py

```diff
@@ -0,0 +1,296 @@
+import pandas as pd
+import numpy as np
+import random
+import matplotlib.pyplot as plt
```

Comment:
I'm getting an error when I run the tests taht matplot lib isn't installed. You should run `poetry add matplotlib` in the command line to install it.